Examples/MAX78000/CNN/kws20_demo/main.c

```diff
@@ -837,16 +845,19 @@ uint8_t check_inference(q15_t *ml_soft, int32_t *ml_data, int16_t *out_class, do
 #else
             MXC_TFT_ClearScreen();
             memset(buff, 32, TFT_BUFF_SIZE);
-            TFT_Print(buff, 20, 30, font_2,
-                      snprintf(buff, sizeof(buff), "%s (%0.1f%%)", keywords[max_index],
-                               (double)100.0 * max / 32768.0));
+            if (max_index == 20 || *out_prob <= INFERENCE_THRESHOLD)
+                TFT_Print(buff, 20, 30, font_2, snprintf(buff, sizeof(buff), "Unknown"));
+            else
+                TFT_Print(buff, 20, 30, font_2,
+                          snprintf(buff, sizeof(buff), "%s (%0.1f%%)", keywords[max_index],
+                                   (double)100.0 * max / 32768.0));
             TFT_Print(buff, 1, 50, font_1,
                       snprintf(buff, sizeof(buff), "__________________________ "));
-            TFT_Print(buff, 1, 80, font_1, snprintf(buff, sizeof(buff), "Top classes:"));
+            //TFT_Print(buff, 1, 80, font_1, snprintf(buff, sizeof(buff), "Top classes:"));
         } else {
-            TFT_Print(buff, 20, 80 + 20 * top, font_1,
-                      snprintf(buff, sizeof(buff), "%s (%0.1f%%)", keywords[max_index],
-                               (double)100.0 * max / 32768.0));
+            //TFT_Print(buff, 20, 80 + 20 * top, font_1,
```

Comment:
Delete commented code?

Reply:
I'd rather keep it in case somebody wants to see top 5 classes.